### PR TITLE
add spot and font fix

### PR DESCRIPTION
**Pull request by StrongCurrent**
Here is my code for the second users story. The font was still embedded incorrectly, which i have also fixed. I look forward to your feedback. Thank you very much

### CODE REVIEW

File: components/AddSpot/index.js [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+import { useRouter } from "next/router";
+import mockData, { addSpotData } from "../../lib/mock-data";
+
+export default function NewSpotForm() {
+  const [spotName, setSpotName] = useState("NAME THIS SPOT HERE");
+  const [longitude, setLongitude] = useState("");
+  const [latitude, setLatitude] = useState("");
+  const [isFocused, setIsFocused] = useState(false);
+  const [isNameError, setIsNameError] = useState(false);
+  const [isDuplicateNameError, setIsDuplicateNameError] = useState(false);
```
Comment (HannesOster) [inline components/AddSpot/index.js:12]: Wouldn´t it be possible to comprimise this states? 
i guess you could use an array of objects instead as initialvalue

File: components/AddSpot/index.js [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  margin-bottom: 20px;
+`;
+
+const AddSpotLabel = styled.label`
+  display: block;
+  font-size: 1.2rem;
+  color: #2f6673;
+  list-style-type: none;
+  margin-right: 10px;
+`;
```
Comment (HannesOster) [inline components/AddSpot/index.js:195]: Styles like these you could export in a styles.js

**Review by HannesOster — APPROVED** [review]
Hey Martin, 
your code looks good, there just were 2 things i would suggest you to change for your next PR. The comment for the states could be unnecessary if you integrate a database later anyway.. 
See you later :)

File: components/AddSpot/index.js (hunk 2 above)
Comment (StrongCurrent) [reply components/AddSpot/index.js]: @HannesOster, thank you for your approval and your feedback. i have adjusted the necessary things.

**Pull request merged**